src/domain/app/routes/constants.ts

```diff
@@ -1,16 +1,15 @@
 import { MAPPED_PLACES } from '../../eventSearch/constants';
 
 // Remember also update static urls to updateSitemap
-
 export type ROUTE_TYPES = {
   ABOUT: '/about';
   ACCESSIBILITY: '/accessibility';
   COLLECTION: '/collection/:slug';
   COLLECTIONS: '/collections';
   EVENTS: '/events';
-  EVENT: '/event/:id';
+  EVENT: '/events/:id';
```

Comment:
This change can cause some problems if there are already some shared events e.g. in facebook. After this change they will open 404 page. You could add redirect from '/event/:id' to '/events/:id' to avoid this problem. Of course the problem is only temporary

Reply:
good to know, maybe I will add some kind of redirection for now

Reply:
fixed